index.html

```diff
@@ -10,17 +10,54 @@
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
 	</head>
 	<body>
 		<div id="container">
```

Comment:
@filippia The good thing is you're using this div to house the rest of the content on the page. You should further use it to you're advantage by putting the 960px width on it. Remember, when we look at a web page, we see boxes everywhere. Break down you're page into manageable "boxes" and work from there.